### PR TITLE
Input

### PR DESCRIPTION
**Pull request by arjun-computer-geek**
input field created with error message.

### CODE REVIEW

File: components/input.css [hunk 1]
```diff
@@ -82,9 +92,63 @@
...
+    display: inline-block;
+}
+
 /* css for mobile responsive */
-@media only screen and (max-width: 600px){
-    .input-field{
+@media only screen and (max-width: 600px) {
+    .input-field {
         padding: 0.5rem;
         margin: 0.25rem;
```
Comment (Shubhamkr2610) [inline components/input.css:153]: Suggestion : you can use px here for less than 1rem values

**Review by Shubhamkr2610 — APPROVED** [review]
code is clean and readable . you can merge it

**Review by Rockyraven — APPROVED** [review]
All goods to me You can merge

File: components/input.css (hunk 1 above)
Comment (arjun-computer-geek) [reply components/input.css:153]: I used to use rem.

**Pull request merged**